packages/thirdweb/src/ThirdWeb.test.ts

```diff
@@ -0,0 +1,267 @@
+import { mint } from './ThirdWeb'
+// import { getFees, mint, simulateMint } from './ThirdWeb'
```

Comment:
cleanup

Reply:
Ive left the commented code so that I can easily run the non-mocked tests when needed.  (the mocked tests are kind of useless 😅)

Wondering if maybe there is a better way to have non-mocked tests to be run locally.